management/src/test/java/org/ehcache/management/providers/statistics/StatsUtil.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.management.providers.statistics;
+
+import org.ehcache.management.ManagementRegistryService;
+import org.terracotta.management.model.context.Context;
+import org.terracotta.management.model.context.ContextContainer;
+import org.terracotta.management.model.stats.AbstractStatisticHistory;
+
+/**
+ *
+ *
+ */
+public class StatsUtil {
+
+  public static boolean isHistoryReady(AbstractStatisticHistory counterHistory, Double defaultValue)
+  {
```

Comment:
shoot i thought I had fixed all those.  I just updated it to the correct format.